<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_VideoFrame</class>
 <widget class="QWidget" name="Form_VideoFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoFrame</string>
  </property>
  <layout class="QGridLayout" name="pGridLayout_VideoFrame">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="pGridLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="1">
      <widget class="QFrame" name="pFrame_TestMode">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="pGridLayout_TestMode">
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="pLabel_TestMode_Title">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Тестовый режим
(Test Mode)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="pCheckBox_TestModeOnOff">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Вкл/Выкл (On/Off)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="pGridLayout_Zoom">
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="6" column="1">
        <widget class="QLineEdit" name="pLineEdit_VideoViewingFPS">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>60</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="pLabel_StartRecordTime">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="pLineEdit_HorizontalZoomCenter">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1032</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="pLineEdit_VerticalZoomCenter">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>772</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="pLabel_Zoom_Title">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Увеличение (Zoom)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="pCheckBox_VideoRecording">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Video Recording</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="pLabel_RecordTime">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="pLabel_VideoViewingFPS_Title">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Частота кадров (FPS)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="pCheckBox_VideoViewing">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Video Viewing</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QGroupBox" name="pGroupBox_Compression">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Сжатие (Compression)</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="pGridLayout_GroupBox_Zoom_2">
            <property name="spacing">
             <number>5</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="pRadioButton_Compression_No">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Нет (No)</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="pRadioButton_Compression_Yes">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Есть (Yes)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="pComboBox_CodecChoice">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="pGroupBox_ZoomFactor">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Кратность (Factor)</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="pGridLayout_GroupBox_Zoom">
            <property name="spacing">
             <number>5</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="pRadioButton_ZoomFactor_x1">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>x1</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="pRadioButton_ZoomFactor_x4">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>x4</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="pRadioButton_ZoomFactor_x8">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>x8</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="pRadioButton_ZoomFactor_x2">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>x2</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QRadioButton" name="pRadioButton_ZoomFactor_x16">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>x16</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="pLabel_MouseCursorPosition">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>X = ; Y = </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QLabel" name="pLabel_VideoRecordFileName">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>D:WorkDirBrigRecord_001.avi</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QLineEdit" name="pLineEdit_VideoRecordFileName_Manual">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>D:\WorkDir\Brig\Record_001.avi</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QGroupBox" name="pGroupBox_VideoRecordFileName">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Имя файла (VideoRecordFileName)</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="pGridLayout_GroupBox_Zoom_3">
            <property name="spacing">
             <number>5</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="pRadioButton_VideoRecordFileName_Manual">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Ручное (Manual)</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="pRadioButton_VideoRecordFileName_Auto">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Автоматическое (Auto)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QLineEdit" name="pLineEdit_VideoRecordFileName_Auto">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>D:\WorkDir\Brig\Record_</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QGraphicsView" name="pGraphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1920</width>
         <height>1080</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1920</width>
         <height>1080</height>
        </size>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::NoDrag</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
